addons/test_website/tests/test_performance.py

```diff
@@ -7,4 +7,4 @@
 class TestPerformance(UtilPerf):
     def test_10_perf_sql_website_controller_minimalist(self):
         url = '/empty_controller_test'
-        self.assertEqual(self._get_url_hot_query(url), 3)
+        self.assertEqual(self._get_url_hot_query(url), 1)
```

Comment:
As per the other comment about `_get_cached`, this will become 0 query once `_get_cached()` is restored :partying_face: